bin/batch-bag

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [[ -z "$2" ]]; then
+  echo "Usage: batch-bag <audio|video> <directory> [additional_directory...]"
+  exit 1
+fi
+
+if [[ -n "$CHIPMUNK_HOME" ]]; then
+  if [[ -x "$CHIPMUNK_HOME/bin/makebag" ]]; then
+    MAKEBAG="$CHIPMUNK_HOME/bin/makebag"
+  else
+    echo "If CHIPMUNK_HOME is set, it must point to a chipmunk-client directory with bin/makebag in it."
+    exit 1
+  fi
+elif [[ ! $(which makebag) ]]; then
+  echo "The chipmunk-client bin directory (with makebag and upload) must be on the PATH or you must set CHIPMUNK_HOME to the client directory."
+  exit 1
+else
+  MAKEBAG=$(which makebag)
+fi
+
+if [[ -z "$CHIPMUNK_LOGS" ]]; then
+  echo "You must set CHIPMUNK_LOGS to a directory that is writeable for log files."
+  exit 1
+fi
+
+mkdir -p "$CHIPMUNK_LOGS"
+
+CONTENT_TYPE=$1
+shift
+
+LOGFILE="$CHIPMUNK_LOGS/batch-$(date +"%Y%m%d%H%M%S").log"
+
+echo "Making batch of bags of type: $CONTENT_TYPE"
+echo "Logging to: $LOGFILE"
+echo "----"
+
+for file in "$@"; do
+  barcode=$(basename $file)
+  directory=$(dirname $file)
+  output=$directory/bagged/$(basename $file)
+  if [[ ! $barcode =~ ^39015.* ]]; then
+    echo "Skipping apparent non-barcode $barcode";
```

Comment:
Is this just to prevent problems arising from running `batch-bag audio *`?

Reply:
I think we know that these things will "always" follow this barcode pattern. It might be excessive, but it is unchanged from the original.